task1.2.js

```diff
@@ -0,0 +1,31 @@
+import CSVToJSON from "csvtojson";
+import fs from "fs";
+
+const readStream = fs.createReadStream('./csv/csvtojson.csv')
+const writeStream = fs.createWriteStream('./csv/users.txt')
+
+writeStream.on('error', err => {
+    console.log('File write Error :: ', err)
+})
+
+// read line by line
+CSVToJSON()
```

Comment:
Use the pipe also (file is not loaded fully in the RAM(pipeline method https://nodejs.org/api/stream.html#stream_stream_pipeline_streams_callback))